wiki/Client/Beatmap_editor/Menu/de.md

```diff
@@ -1,178 +1,92 @@
-| AiMod öffnen (`Strg` + `Shift` + `A`) | AiMod öffnen. |
-
-### Befehle zum Verpacken von Beatmaps
-
-| Name | Beschreibung |
-| :-- | :-- |
-| Beatmap hochladen ... (`Strg` + `Shift` + `U`) | Lädt die Beatmap in die Sektion [WIP](https://osu.ppy.sh/community/forums/10) hoch. Du musst mit deinem Account auf der Webseite angemeldet sein, um das Beatmap-Forum sehen zu können. |
-| Paket exportieren ... | Exportiert die Beatmap als `.osz`. Die Datei wird zum Ordner "Exports" gesendet und der Ordner wird direkt danach geöffnet. |
-| Map-Paket entpacken | ? |
-| Importieren von ... | **bms/bme:** (nur für osu!mania) Öffnet .bms/.bme Dateien, um sie als osu!mania-Schwierigkeitsstufe zu importieren. |
-
-### Befehle für Programmierer
-
-| Name | Beschreibung |
-| :-- | :-- |
-| Songs-Ordner öffnen | Öffnet den Song-Ordner, welcher sich im Verzeichnis osu!/Songs befindet. |
-| `.osu` im Editor öffnen | Öffnet die Schwierigkeitsstufe im Texteditor. |
-| `.osb` im Editor öffnen | Öffnet die Storyboard-Datei im Texteditor. |
-
-### Befehl für den Notausstieg
-
-| Name | Beschreibung |
-| :-- | :-- |
-| Beenden... (`Esc`) | Beatmap-Editor verlassen. Du wirst möglicherweise gefragt, ob du kürzliche Änderungen speichern möchtest. *Ignoriert* manchmal Änderungen, die im Design-Tab gemacht wurden. |
+| Schwierigkeitsstufe öffnen ... | **Zum Editieren:** Wechsle schnell zwischen den Schwierigkeitsstufen. **Zur Referenz:** Lege die gewählte Schwierigkeitsstufe über die (aktuell) geöffnete (nur [osu!mania](/wiki/Game_mode/osu!mania)). |
+| Speichern (`Strg` + `S`) | Speichert alle Änderungen (Dateiformate [`.osu`](/wiki/Client/File_formats/osu_(file_format)) und [`.osb`](/wiki/Client/File_formats/osb_(file_format))). |
+| Neue Schwierigkeitsstufe erstellen ... | Speichere die aktuellen Änderungen dieser Schwierigkeitsstufe als eine neue Stufe. Die alte Schwierigkeitsstufe bleibt beim erneuten Laden in ihrem zuletzt gespeicherten Zustand erhalten. |
+| Zum Gespeicherten zurücksetzen (`Strg` + `L`) | Hebt alle Änderungen der Schwierigkeitsstufe und des Storyboards bis zum letzten Speicherpunkt auf. |
+| Zum Gespeicherten zurücksetzen (Komplett) (`Strg` + `Shift` + `L`) | Hebt alle Änderungen der Schwierigkeitsstufe und des Storyboards bis zum letzten Speicherpunkt auf und lädt alle Dateien neu (Sprites, Soundsamples etc.). |
+| Beatmap testen (`F5`) | [Teste](/wiki/Client/Beatmap_editor/Test_mode) die Beatmap. Aktuelle Änderungen werden für den Test gespeichert. |
+| AiMod öffnen (`Strg` + `Shift` + `A`) | [AiMod](/wiki/Client/Beatmap_editor/AiMod) öffnen. |
+| Beatmap hochladen ... (`Strg` + `Shift` + `U`) | [Lade](/wiki/Beatmapping/Beatmap_submission) die Beatmap in das Forum [Work In Progress](https://osu.ppy.sh/community/forums/10) hoch. |
+| Paket exportieren ... | Exportiert die Beatmap als [`.osz`-Archiv](/wiki/Client/File_formats/osz_(file_format)) zur manuellen Weitergabe. Der Ordner `Exports` wird geöffnet, in dem die Datei abgelegt wird. |
```

Comment:
Weitergabe feels a bit clumsy, but I don't have a better idea..

Reply:
Yeah, among these alternatives, Weitergabe is still the best.. As I said, I don't have a great alternative either